dags/datasets/police-annual-statistical-report.py

```diff
@@ -0,0 +1,223 @@
+# police-annual-statistical-report.py - this file makes multiple DAGs; each is an ETL for a police annual report from AGOL into CKAN
```

Comment:
So, to test this police "multi-dag", I used the replicator to move all the police datasets from PROD to QA, then ran each DAG successfully.

FYI these DAGs run once per year